### ISSUE TITLE
Suggest to add PAC mode

### ISSUE THREAD

**Issue by vidalouiswang** [Jigsaw-Code/outline-client#602]
Hello, thank you for bring us Outline client, it's very useful. I suggest to add PAC mode to the client or let user choose which app should go through encrypted line, that will be more convenient, thanks!

**Comment by 13025309325** [comment]
We need this, but they don’t seem to add it for a year now

**Comment by cuiqg** [comment]
[为什么到现在都没做? 3年了](https://github.com/orgs/Jigsaw-Code/projects/1#card-53762659)

**Comment by DDDOH** [comment]
PAC is needed for saving data usage

**Comment by daniellacosse** [comment]
Related to https://github.com/Jigsaw-Code/outline-client/issues/887?

**Comment by Kayx23** [comment]
> Related to #887?

Yes it is related. I suggest we consolidate these issues and discuss in one issue/conversation, following up there...

**Comment by Kayx23** [comment]
> 为什么到现在都没做? 3年了

it is now 2023

**Comment by edica02** [comment]
and we desperately need a PAC mode!
